server/src/sockets/chess.socket.js

```diff
@@ -23,9 +26,27 @@ const disconnectPlayers = (socket, io) => {
 };
 
 const chessSocket = (server) => {
-  const io = IO(server);
+  const io = IO(server, {
+    handlePreflightRequest(req, res) {
+      const headers = {
+        'Access-Control-Allow-Headers': 'x-auth-token',
+        'Access-Control-Allow-Origin': 'http://localhost:3000',
```

Comment:
Assign value to "Access-Control-Allow-Origin" from environment variables.

Reply:
done